string ip_addr
string net_mask
string gw_addr